hipcheck/src/unstable.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{
+	cli::{CliConfig, UnstableArgs},
+	session::session::Check,
+	shell::{ColorChoice, Output},
+};
+use std::io::Write;
+use std::{io, time::Instant};
+use tempdir::TempDir;
+
+/// Handle unstable commands.
+pub fn main(args: UnstableArgs, config: &CliConfig) -> io::Result<()> {
```

Comment:
I would stylistically opt for a different name, such as `run` or `run_unstable`, but not a big deal